include/library/spdm_crypt_lib.h

```diff
@@ -404,6 +412,24 @@ bool libspdm_asym_get_public_key_from_der(uint32_t base_asym_algo,
  **/
 void libspdm_asym_free(uint32_t base_asym_algo, void *context);
 
+
```

Comment:
Everything in this header, besides the `SIG_SIZE` macros, can go into `internal/libspdm_crypt_lib.h`.

Reply:
@steven-bellock I think I only added the *SIG_SIZE macros for the PR.  Perhaps other general cleanup changes should go into a different PR.